tests/Fixer/PhpUnit/PhpUnitOrderedAnnotationsFixerTest.php

```diff
@@ -0,0 +1,954 @@
+                     */
+                    class FooTest extends \PHPUnit_Framework_TestCase {}
+                ',
+            ],
+            'data provider' => [
+                '<?php
+                    class FooTest extends \PHPUnit_Framework_TestCase
+                    {
+                        /**
+                         * @uses Bar
+                         * @dataProvider provide
+                         * @uses Foo
+                         */
+                        public function testMe() {}
+                    }
+                ',
+                '<?php
+                    class FooTest extends \PHPUnit_Framework_TestCase
+                    {
+                        /**
+                         * @uses Foo
+                         * @dataProvider provide
+                         * @uses Bar
+                         */
+                        public function testMe() {}
+                    }
+                ',
+            ],
+        ];
+    }
+
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixWithMultipleConfiguredAnnotationsCases
+     */
+    public function testFixWithMultipleConfiguredAnnotations($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'annotations' => [
+                'covers',
+                'uses',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithMultipleConfiguredAnnotationsCases()
+    {
+        return [
+            'skip on 1 or 0 occurrences' => [
+                '<?php
+                    class FooTest extends \PHPUnit_Framework_TestCase {
+                        /**
+                         * @uses Foo
+                         * @covers Baz
+                         * @params bool $bool
+                         * @return void
+                         */
+                        public function testMe() {}
+
+                        /**
+                         * @params bool $bool
+                         * @return void
+                         */
+                        public function testMe2() {}
+                    }
+                ',
+            ],
+            'base case' => [
+                '<?php
+                    /**
+                     * @uses Bar
+                     * @uses Foo
+                     * @covers Baz
+                     * @covers Qux
+                     */
+                    class FooTest extends \PHPUnit_Framework_TestCase {}
+                ',
+                '<?php
+                    /**
+                     * @uses Foo
+                     * @uses Bar
+                     * @covers Qux
+                     * @covers Baz
+                     */
+                    class FooTest extends \PHPUnit_Framework_TestCase {}
+                ',
+            ],
+            'preserve positions if other docblock parts are present' => [
```

Comment:
This test case currently fails.

🤔

Reply:
I think we (you ;) ) should not need to trouble ourselves to much with `@author` annotations sticking to a piece of PHPDoc. 
For such details I think people can better refer to their VCS and only a list of authors per file can be handy. However, if they really want to keep the authors sticky to the rest of the text than they can remove the `@author` annotation from the configuration of the rule.

Reply:
@SpacePossum 

Fixed it, had something messed up in my expectations!

👍